test/parallel/test-net-connect-options-ipv6.js

```diff
@@ -81,7 +79,7 @@ function tryConnect() {
       if (host)
         tryConnect();
       else {
-        common.skip('no IPv6 localhost support');
+        common.printSkipMessage('no IPv6 localhost support');
```

Comment:
also I think this is a `skip` (exiting will close the server, and also you can flip the order, `server.close` then skip`